src/utils/HyperSwitchUtils.res

```diff
@@ -38,6 +38,11 @@ let delay = ms =>
 
 let checkIsInternalUser = roleId => {
   open UserManagementUtils
-  roleId->stringToVariantMapper == InternalViewOnly ||
-    roleId->stringToVariantMapper == InternalAdmin
+  roleId->stringToVariantMapperInternalUser == InternalViewOnly ||
+    roleId->stringToVariantMapperInternalUser == InternalAdmin
+}
```

Comment:
let checkIsInternalUser = roleId => {
  let userType = roleId->UserManagementUtils.stringToVariantMapperInternalUser
  userType == InternalViewOnly || userType == InternalAdmin
}

Change the var names according to you don't have much context what it is been used for